TOOLS/ce/src/main/scala/org.opalj.ce/SubclassExtractor.scala

```diff
@@ -0,0 +1,48 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package ce
+
+import java.net.URL
+import scala.collection.mutable
+
+import org.opalj.br.ClassHierarchy
+import org.opalj.br.ObjectType
+import org.opalj.br.ObjectType.unapply
+import org.opalj.br.analyses.Project
+
+/**
+ * The class subclassExtractor is a wrapper class around the bytecode representation project.
+ * It will fetch all subclasses from the opal project and prepares the bytecode hierarchies for querying.
+ * @param f accepts any initialized FileLocator.
+ * @param pathWildcard accepts a Wildcard that must be present in every .jar files file name in order to be used for hierarchy extraction
+ */
+class SubclassExtractor(val f: FileLocator, pathWildcard: String) {
+    var classHierarchies: ClassHierarchy = null
```

Comment:
Why is it called `classHierarchies` when it is just a single one?

Reply:
Fixed